rds-instance-full/main.tf

```diff
@@ -43,9 +43,9 @@ module "rds_security_group" {
 module "custom_security_group" {
   source = "terraform-aws-modules/security-group/aws"
 
-  create = "${var.rds_sg == "" ? 0 : 1 }"
+  create = "${var.custom_sg_id == "" ? 0 : 1 }"
 
-  name        = "${local.identifier}-rds-ecs"
+  name        = "${local.identifier}-rds-custom"
   description = "Security group with RDS ports open for ECS"
```

Comment:
"ECS" => "Security group with RDS ports open for a custom security group"

Reply:
I missed that one, thanks!